backend/.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+      {
+        "type": "node",
+        "request": "attach",
+        "name": "Remote Debug NestJS app in Docker",
+        "port": 9999,
+        "localRoot": "${workspaceFolder}",
+        "remoteRoot": "/app",
+        "restart": true,
+        "sourceMaps": true,
+        "outFiles": [
+          "${workspaceFolder}/dist/**/*.js"
+        ]
+      }
+    ]
+  }
```

Comment:
Added launch.json to configure a remote debugger to the NestJS backend in VS code. 

I can remove this file from the commit if requested :)